src/REGothEngine.cpp

```diff
@@ -17,48 +18,23 @@ REGothEngine::~REGothEngine()
   shutdown();
 }
 
-bs::Vector<bs::String> REGothEngine::getVdfsPackagesToLoad(const bs::Path& dataDirectory)
+bs::Vector<bs::Path> REGothEngine::getVdfsPackagesToLoad(const bs::Path& dataDirectory)
```

Comment:
Sorry deleted my previous comment. The parameter `dataDirectory` seems to be obsolete now.

Reply:
Removed it in 59b0320.